test/cpp/json-parse.cpp

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2017 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
+
+#include <nan.h>
+
+NAN_METHOD(Parse) {
+  Nan::JSON NanJSON;
+
+  Nan::MaybeLocal<v8::String> inp = Nan::To<v8::String>(info[0]);
+
+  if (inp.IsEmpty()) {
+    info.GetReturnValue().Set(Nan::Undefined());
```

Comment:
Better to remove them, as they are not needed. We don't explicitly set the return value to `Undefined` anywhere else, so it could be confusing to suddenly see it here.